ethcore/res/contracts/authority_round_random.json

```diff
@@ -0,0 +1,158 @@
+[{
+		"constant": true,
+		"inputs": [],
+		"name": "currentRandom",
```

Comment:
@afck Let's remove `currentRandom` from the Random contract ABI since we don't use that (and we replaced that with [`uint256 public currentSeed`](https://github.com/poanetwork/posdao-contracts/blob/claim-reward/contracts/RandomAuRa.sol#L28)).

Reply:
Done.